test/other/metadce/test_metadce_files_wasmfs.size

```diff
@@ -1 +1 @@
-47462
+51960
```

Comment:
Quite a large increase compared to the actual change in this PR... looks like strings operations mostly? Not sure we can avoid that.